src/storage/cdb/cdb_ccommon/src/lib.rs

```diff
@@ -0,0 +1,8 @@
+pub mod bindings {
+    #![allow(non_upper_case_globals)]
+    #![allow(non_camel_case_types)]
+    #![allow(non_snake_case)]
+    #![allow(dead_code)]
+    include!(concat!(env!("OUT_DIR"), "/bindings.rs"));
+}
```

Comment:
This block, cargo-culted from the bindgen user guide, pulls all of the generated types in to the 'bindings' mod